velox/exec/Driver.h

```diff
@@ -403,13 +403,15 @@ class Driver : public std::enable_shared_from_this<Driver> {
   friend struct DriverFactory;
 };
 
+#ifdef VELOX_ENABLE_BACKWARD_COMPATIBILITY
```

Comment:
Unrelated change?

Reply:
I have separated these changes in a separate pr.